tests/thread_basic/tests/01-run.py

```diff
@@ -12,6 +12,7 @@
 sys.path.append(os.path.join(os.environ['RIOTBASE'], 'dist/tools/testrunner'))
 import testrunner
 
+
```

Comment:
unrelated and not needed?

Reply:
Well, that just a fix for a small pep8

Reply:
I think that fixing pep8 could be done in a different commit.

Reply:
Ok ;)

Reply:
Done